docs/reference/rack-parameters.md

```diff
@@ -124,6 +124,20 @@ Encrypt secrets with KMS.
 
 Existing VPC ID (if blank, a VPC will be created).
 
+
+### HighAvailability
+
+<div class="alert alert-warning">
+This parameter cannot be changed after the rack is created.
+</div>
+
+Whether to enable High Availability mode. This ensure proper resources redundancy to mitigate system failures.
```

Comment:
Whether or not to enable..... Choose between failure resiliency and cost efficiency.